### PR TITLE
Bug fixing v2

### PR DESCRIPTION
**Pull request by linuskar**
Updated Readme
Added stats for owned maps
Removed unused/old scenes
Fixed PlayMode tests breaking from saving player position
Added documents folder

### CODE REVIEW

File: Documents/TeamReflections/Sprint 1 reflection.pdf
Comment (Caramajau) [inline Documents/TeamReflections/Sprint 1 reflection.pdf:1]: The reflections don't need to be in the repo

File: README.md [hunk 1]
```diff
@@ -1,8 +1,48 @@
...
+### Documents
+Go to [Documents](https://github.com/Oskmey/-Agile-software-project-management/tree/bug_fixing_v2/Documents) to see some our documents.
+
+### Wiki
+We have a [wiki page](https://github.com/Oskmey/-Agile-software-project-management/wiki) for the GitHub repository. There you can find additonal information.
+
+## Authors 
+* Aron Lund - [AronLundh](https://github.com/AronLundh)
+* David Memedov – [Caramajau](https://github.com/Caramajau)
+* Edgar Young - [FIger-op](https://github.com/Figer-op)
```
Comment (Caramajau) [inline README.md:30]: Minor spelling mistake, right now it is FIger, when it should be Figer

**Review by Caramajau — CHANGES_REQUESTED** [review]
failing_tests

All tests don't pass anymore, I believe it is because some scenes were deleted?

**Comment by linuskar** [comment]
> failing_tests
> 
> All tests don't pass anymore, I believe it is because some scenes were deleted?

Make sure the test scenes are in the build settings

**Review by Caramajau — APPROVED** [review]

**Review by Figer-op — APPROVED** [review]
ok

**Pull request merged**